server/supabase/migrations/20240621172426_timestamptz.sql

```diff
@@ -0,0 +1,17 @@
+alter table queries alter created_at type timestamptz using created_at at time zone 'utc';
```

Comment:
just to make sure - do we generate the timestamp on DB side for all of these columns during insert/update?

Reply:
yep!